src/pages/MyAccount/index.tsx

```diff
@@ -1,46 +1,141 @@
+import { Box, Button, HStack, Image, Text, useBreakpointValue, VStack } from '@chakra-ui/react';
 import styled from '@emotion/styled';
+import { useEffect, useState } from 'react';
 
-import { Button } from '@/components/common/Button';
-import { Spacing } from '@/components/common/layouts/Spacing';
+import { fetchInstance } from '@/api/instance';
 import { useAuth } from '@/provider/Auth';
-import { RouterPath } from '@/routes/path';
+import { breakpoints } from '@/styles/variants';
 import { authSessionStorage } from '@/utils/storage';
 
-export const MyAccountPage = () => {
+interface WishlistItem {
+  id: number;
+  product: {
+    id: number;
+    name: string;
+    price: number;
+    imageUrl: string;
+  };
+}
+
+export const MyAccount = () => {
   const authInfo = useAuth();
+  const [wishlist, setWishlist] = useState<WishlistItem[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
 
-  const handleLogout = () => {
-    authSessionStorage.set(undefined);
+  useEffect(() => {
+    const fetchWishlist = async () => {
+      if (authInfo) {
+        setLoading(true);
+        try {
+          const response = await fetchInstance.get(
+            '/api/wishes?page=0&size=10&sort=createdDate,desc',
```

Comment:
```suggestion
            '/api/wishes', { params: { page: 0, size: 10, sort: 'createdDate,desc' } }
```
ref. https://axios-http.com/kr/docs/example